ContactTableViewController.m

```diff
@@ -0,0 +1,132 @@
+//
+//  ContactTableViewController.m
+//  FilterSearchAttempt
+//
+//  Created by Lizzy Randall on 5/5/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import "ContactTableViewController.h"
+#import "Contact.h"
+
+@interface ContactTableViewController ()
+
+@end
+
+@implementation ContactTableViewController
+
+@synthesize contactArray;
+@synthesize filteredContactsArray;
+@synthesize contactSearchBar;
+@synthesize searchController;
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.searchController = [[UISearchController alloc] initWithSearchResultsController:nil];
+    self.searchController.searchResultsUpdater = self;
+    self.searchController.dimsBackgroundDuringPresentation = NO;
+    self.searchController.searchBar.delegate = self;
+    
+    self.tableView.tableHeaderView = self.searchController.searchBar;
+    
+    self.definesPresentationContext = YES;
+    
+    [self.searchController.searchBar sizeToFit];
+    
+    contactArray = [NSArray arrayWithObjects:
```

Comment:
You should always use self. with a classes property unless you are overriding a setter (in the case of a setter you reference the object's ivar via _contactArray).
Once you start using only @property statements for property creation the IDE will remind you to change this to self.contactArray